JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/ViewModels/HomeViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Threading.Tasks;
+using JhipsterXamarin.Services;
+using MvvmCross.Commands;
+using MvvmCross.Navigation;
+using MvvmCross.ViewModels;
+
+namespace JhipsterXamarin.ViewModels
+{
+    public class HomeViewModel : BaseViewModel
+    {
+        private readonly IMvxNavigationService _navigationService;
+        public IMvxAsyncCommand ShowEntitiesViewModelCommand { get; }
+        public IMvxAsyncCommand LoginViewModelCommand { get; }
+        public IMvxAsyncCommand ShowMenuViewModelCommand { get; }
+
+        public HomeViewModel(IMvxNavigationService navigationService)
```

Comment:
please check if this corresponds to the viewmodel life cycle 
https://www.mvvmcross.com/documentation/fundamentals/viewmodel-lifecycle

Reply:
It should I guess